fhir-model/src/main/java/com/ibm/fhir/model/path/FHIRPathBaseVisitor.java

```diff
@@ -1,498 +1,301 @@
-/*
- * (C) Copyright IBM Corp. 2019
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-
+// Generated from FHIRPath.g4 by ANTLR 4.7.2
 package com.ibm.fhir.model.path;
```

Comment:
maybe add our `@Generated` annotation?

Reply:
I'm going to leave the ANTLR generated code as is.